docker/docker_client.go

```diff
@@ -394,7 +395,7 @@ func (c *dockerClient) ping() (*pingResponse, error) {
 		if isV1 {
 			err = ErrV1NotSupported
 		} else {
-			err = fmt.Errorf("pinging docker registry returned %+v", err)
+			err = errorspkg.Wrap(err, "pinging docker registry returned")
```

Comment:
So in our case we check if a `url.Error` is returned and make some assumptions on it. Without this change, when the SSL validation fails for example, client returns an `url.Error` that gets merged into this generic pinging `error`, making it hard to distinguish it.